AzureSignTool/Program.cs

```diff
@@ -206,44 +239,48 @@ static int Main(string[] args)
                           {
                               return state;
                           }
-                          using (var logger = LoggerServiceLocator.Current.Scoped())
+                          var loopLogger = serviceProvider.GetRequiredService<ILogger<Program>>();
+                           
+                          using (var loopScope = loopLogger.ItemIdScope(Interlocked.Increment(ref itemScope)))
```

Comment:
Not sure if that's the right way to use scope. It appears to mutate the logger instance rather than return a new logger, which is why I have the previous like to get a new logger instance.